### ISSUE TITLE
bug: detectZodSchemaRootNode doesn't detect named `z` import

### ISSUE THREAD

**Issue by marcalexiei** [marcalexiei/eslint-plugin-zod-x#174]
### 🧩 Description

Zod namespace is not recognised when imported via named import

```ts
import { z } from 'zod';
const schema = z.string();
```

### ✒️ Additional context

- https://github.com/marcalexiei/eslint-plugin-zod-x/issues/172
